cores/arduino/stm32/twi.c

```diff
@@ -193,7 +193,7 @@ void i2c_custom_init(i2c_t *obj, i2c_timing_e timing, uint32_t addressingMode, u
   handle->Init.OwnAddress2     = 0xFF;
   handle->Init.AddressingMode  = addressingMode;
   handle->Init.DualAddressMode = I2C_DUALADDRESS_DISABLE;
-  handle->Init.GeneralCallMode = I2C_GENERALCALL_DISABLE;
+  handle->Init.GeneralCallMode = I2C_GENERALCALL_ENABLE;
```

Comment:
Does it make sense as well if the STM32 is master ? or shouldn't we enable this feature only if master == false ?

Reply:
I think this always should be enabled, because in multimaster mode, the masters can also act as slaves. I think this needs to be tested with 1 device doing lots of general calls and another device in master mode, and making sure that every general call is received.

Reply:
Ok. but as you mentioned this requires a good testing session. An alternative would be to provide an option in the constructor to let application decide whether to answer General Calls or not.  up to @fpistm

Reply:
Reading some docs about general call, it's not so easy to tell that it should be enabled by default.
What I suggest is to provide a way to enable it if user really want to use it. This way, there is no regression risks.